WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -344,7 +344,7 @@ def post_results
         end
       end
       unless comp.results_posted?
-        comp.update!(results_posted_at: Time.now)
+        comp.update!(results_posted_at: Time.now, results_posted_by: current_user.id)
```

Comment:
(as above) It would be nice to motivate this change with a corresponding change to spec/controllers/competitions_controller_spec.rb. It should be a pretty straightforward change to one of the existing tests.

Reply:
And same here: I'm modifying one of the existing tests to add this.